src/main/webapp/app/overview/course-conversations/dialogs/conversation-detail-dialog/tabs/conversation-info/conversation-info.component.scss

```diff
@@ -14,21 +17,18 @@
                 }
             }
 
-            &.interactive {
-                &:hover {
-                    .action-button {
-                        visibility: visible;
-                    }
-                }
-            }
-
             .wide-button {
                 width: 50%;
             }
 
             textarea {
                 min-height: 5rem;
                 resize: none;
+                border-radius: 16px;
```

Comment:
All other areas of Artemis only use rectangular elements. (Except lectures? IIRC there either was a goal to remove the rounded elements there as well, or it has been done already.)
```suggestion
```

Reply:
Yes, please be consistent regarding the style. CC @bassner

Reply:
The discussion was that rectangular looks too much like normal textfields. @TimOrtel mentioned this.

Reply:
I mean if you want I can make it rectangular again :D

Reply:
I remember that in #5595 the border radius was removed from all components.

Reply:
@ge65cer removed the rounding in 53723110e2bbf839f157663453c6cdf48f23f0ff